packages/cli/src/commands/setup/auth/auth.js

```diff
@@ -95,40 +99,52 @@ const objectToComponentProps = (obj, options = { exclude: [] }) => {
 
 // returns the content of App.{js,tsx} with <AuthProvider> added
 const addWebRender = (content, authProvider) => {
-  const [_, indent, redwoodApolloProvider] = content.match(
-    /(\s+)(<RedwoodApolloProvider>.*<\/RedwoodApolloProvider>)/s
-  )
-
-  const redwoodApolloProviderLines = redwoodApolloProvider
+  const [
+    _,
+    newlineAndIndent,
+    redwoodProviderOpen,
+    redwoodProviderChildren,
+    redwoodProviderClose,
+  ] = content.match(/(\s+)(<RedwoodProvider.*?>)(.*)(<\/RedwoodProvider>)/s)
```

Comment:
Could you also add a test for this? Like add another fixture that doesn't use the RedwoodApolloProvider